dashboard/src/components/DeploymentForm.tsx

```diff
@@ -358,6 +358,11 @@ export default function DeploymentForm({
                                     required
                                     name={['targets', idx, 'config', 'enable_ingress']}
                                     label={t('cluster external access')}
+                                    caption={
+                                        target.config?.enable_ingress
+                                            ? t('accessible from outside the cluster')
+                                            : t('not accessible from outside the cluster')
```

Comment:
simply say "Enabled ℹ️" / "Disabled ℹ️", and for the "info" icon, we can do a on-hover pop-up and tell user more details:

* Public endpoint access enabled. Yatai will make the deployment endpoint accessible from outside of the cluster and create necessary ingress resources in the target cluster.
* Public endpoint access disabled. Yatai will create a Kubernetes Service for the deployment which can only be accessed from within the target cluster. If you have direct access to the cluster, you may use `kubectl port-forward` for testing the endpoint locally.